### ISSUE TITLE
Please please, can you write a script to run on Google Collab?

### ISSUE THREAD

**Issue by AIhasArrived** [Spycsh/xtalker#11]
Can you?
I cannot run sadtalker because it has a virus apparently.
I can only run it on google collab.
Could someone help me write the collab code?

**Comment by Spycsh** [comment]
Hi @AIhasArrived ,this repo is not built for Google colab because colab usually has limited cpu resources. Specifically, this repo accelerate SadTalker on many-core Xeon CPU. If you have any issue on the original SadTalker regarding virus or installation, please raise that to the original repo.

**Comment by AIhasArrived** [comment]
Hello @Spycsh , even Google Collab Pro +  Would not run it? https://colab.research.google.com/signup

**Comment by Spycsh** [comment]
@AIhasArrived , I'm afraid Collab Pro+ only upgrade GPUs.

**Comment by AIhasArrived** [comment]
> @AIhasArrived , I'm afraid Collab Pro+ only upgrade GPUs.

Yeah, that's what I figured afterwards, and I think they don't even upgrade GPU that much, they just give you way more units of computations without necessarily giving you the best ones.
